src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.codeu.data.Datastore;
+
+/**
+ * Handles fetching all users for the community page.
+ */
+@WebServlet("/user-list")
+public class UserListServlet extends HttpServlet {
+  private Datastore datastore = new Datastore();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+    response.getOutputStream().println(datastore.getUsers().toString());
```

Comment:
Lovely 👍 it's always better to test what you have and not just blindly follow the instructions !